src/test/java/uk/gov/pay/api/it/GetDirectDebitEventsTest.java

```diff
@@ -42,19 +42,19 @@
             config("publicAuthUrl", "http://localhost:" + publicAuth.getConfig().getPort() + "/v1/auth"));
 
     private String baseUrl;
-    
+
     @Before
     public void setUp() {
         baseUrl = app.getConfiguration().getBaseUrl();
     }
-    
+
     @Test
     @PactVerification({"direct-debit-connector", "publicauth"})
     @Pacts(pacts = {"publicapi-direct-debit-connector-get-events"})
     @Pacts(pacts = {"publicapi-publicauth"}, publish = false)
     public void getDirectDebitEvents() {
-        String requestPath = "/v1/events?to_date=2018-03-13T10:00:05Z&from_date=2018-03-13T10:00:03Z&display_size=100&page=1&agreement_id=1";
-        
+        String requestPath = "/v1/events?to_date=2018-03-13T10:00:05.000Z&from_date=2018-03-13T10:00:03.000Z&display_size=100&page=1&agreement_id=1";
```

Comment:
Not sure about this one: this in a input date/time (provided by service users) and for those won’t always have millisecond precision

Reply:
Fair point, it slipped through :)